app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    animals = []
```

Comment:
Do you use it somewhere in your code? I suppose not. Remove this class attribute